projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -139,6 +139,7 @@ public static function load_wpcom_user_features() {
 			require_once __DIR__ . '/features/help-center/class-help-center.php';
 		}
 		require_once __DIR__ . '/features/replace-site-visibility/replace-site-visibility.php';
+		require_once __DIR__ . '/features/sync-locale-from-calypso-to-atomic/sync-locale-from-calypso-to-atomic.php';
```

Comment:
Super nit: can we call the feature with a generic name `wpcom-locale`, similar to how we name features below it? I imagine this could be useful if in the future we want to add some other locale-related features.

Reply:
I'll change just the directory's name 👍

Reply:
Oh, that's nice!